examples/registration_form/lib/root/root_component.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:angular/angular.dart';
+import 'package:angular_components/angular_components.dart';
+import 'package:angular_forms/angular_forms.dart';
+import 'package:examples.registration_form/address/address_component.dart';
+
+@Component(
+    selector: 'hello-world',
+    templateUrl: 'root_component.html',
+    directives: [
+      formDirectives,
+      materialInputDirectives,
+      MaterialYesNoButtonsComponent,
+      MaterialSaveCancelButtonsDirective,
+      NgIf,
+      AddressComponent
+    ],
+    providers: [
+      popupBindings,
+    ])
+class RootComponent {
+  @ViewChild('myForm')
+  ControlContainer myForm;
+
+  String firstName;
+  String lastName;
+
+  Person person;
+
+  void onSubmit() {
```

Comment:
I think the pattern that I like to see here instead since you aren't really using myForm elsewhere in the component is to pass in the form to this function using a varref instead of using a ViewChild.

Reply:
Done.